velero-plugin-for-aws/object_store.go

```diff
@@ -76,6 +77,7 @@ type ObjectStore struct {
 	sseCustomerKey       string
 	signatureVersion     string
 	serverSideEncryption string
+	awsTags              string
```

Comment:
instead of using `aws` as a prefix, maybe just use `tags` or `tagging` ?

Reply:
changed it to tagging.